rigs/local-node-rig/profiles/default/config/heft.json

```diff
@@ -1,5 +1,19 @@
 {
-  "$schema": "https://developer.microsoft.com/json-schemas/heft/heft.schema.json",
+  "$schema": "https://developer.microsoft.com/json-schemas/heft/v0/heft.schema.json",
 
-  "extends": "@rushstack/heft-node-rig/profiles/default/config/heft.json"
+  "extends": "@rushstack/heft-node-rig/profiles/default/config/heft.json",
+
+  "phasesByName": {
+    "build": {
+      "tasksByName": {
+        "lint": {
+          "taskPlugin": {
+            "options": {
+              "disableLintConfigSearch": true
```

Comment:
Why isn't the official rig updated to add this setting?  (In a separate comment, I'm suggesting to eliminate this entirely.)

Or at the very least, we need some clear documentation explaining when someone should choose `"disableLintConfigSearch": true` vs `false`.

Reply:
I could update it, but I'd have to do a major version bump of heft-lint-plugin, heft-web-rig, and heft-node-rig, since that _is_ a breaking change It's the behavior we prefer since it avoids footguns, but is nonetheless a breaking change.